integration-cli/docker_cli_rmi_test.go

```diff
@@ -143,25 +112,23 @@ func (s *DockerSuite) TestRmiImgIDForce(c *check.C) {
 	dockerCmd(c, "tag", "busybox-test", "utest:5000/docker:tag4")
 	{
 		imagesAfter, _ := dockerCmd(c, "images", "-a")
-		if strings.Count(imagesAfter, "\n") != strings.Count(imagesBefore, "\n")+4 {
-			c.Fatalf("tag busybox to create 4 more images with same imageID; docker images shows: %q\n", imagesAfter)
-		}
+		c.Assert(strings.Count(imagesAfter, "\n"), checker.Equals, strings.Count(imagesBefore, "\n")+4, check.Commentf("before: %q\n\nafter: %q\n", imagesBefore, imagesAfter))
 	}
 	imgID, err := inspectField("busybox-test", "Id")
-	c.Assert(err, check.IsNil)
+	c.Assert(err, checker.IsNil)
 
 	// first checkout without force it fails
 	out, _, err = dockerCmdWithError("rmi", imgID)
-	if err == nil || !strings.Contains(out, "(must be forced) - image is referenced in one or more repositories") {
-		c.Fatalf("rmi tagged in multiple repos should have failed without force:%s, %v", out, err)
-	}
+	// rmi tagged in multiple repos should have failed without force
+	c.Assert(err, checker.NotNil)
+	// rmi tagged in multiple repos should have failed without force
+	c.Assert(out, checker.Contains, "(must be forced) - image is referenced in one or more repositories", check.Commentf("out: %s; err: %v;", out, err))
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?